cylc/flow/network/schema.py

```diff
@@ -206,7 +219,7 @@ def parse_node_id(item, node_type=None):
 
 # Field args (i.e. for queries etc):
 class SortArgs(InputObjectType):
-    keys = List(String, default_value=['id'])
+    keys = graphene.List(String, default_value=['id'])
```

Comment:
I originally had it `graphene.List(` but it was deemed unnecessary by reviewers.. I guess it can't be helped with `typing`..

Reply:
Yes it's an unfortunate clash. But seeing as `typing` is widely-used now I think it should take precedence